sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -123,6 +123,46 @@ set_target_properties("ur_adapter_level_zero" PROPERTIES
     SOVERSION "0"
 )
 
+if ("cuda" IN_LIST SYCL_ENABLE_PLUGINS)
+  # Build CUDA adapter
+  add_sycl_library("ur_adapter_cuda" SHARED
```

Comment:
@callumfare : do we need a versioning for the CUDA adapter as we have for the L0 adapter (see line 121 above).

Reply:
I'm not sure if there's a need but I've added it anyway to be consistent